sousmot/sousmotapp/static/js/game.js

```diff
@@ -0,0 +1,160 @@
+// Variables declaration
+let userTry = 0; // correspond to grid.x
+let userPosition = 1; // correspond to grid.y
+let word = wordFirstLetter.concat('.'.repeat(wordLength - 1)); // Shared word variable filled with dots and initial letter
+let countDownDate = new Date(end_time * 1000).getTime(); // Date to the end of countdown
+writeWord(); // Write word in grid for first time
+
+/**
+ * Listen to the keydown event user keyboard
+ */
+document.addEventListener('keydown', (event) => {
+    let letter = event.key;
+    addLetterToWord(letter);
+
+    // Uncomment this to display the code value
+    //var code = event.code;
+    //alert(`Key pressed ${letter} \r\n Key code value: ${code}`);
+}, false);
+
+/**
+ * Add a given letter to the grid
+ * @param letter char to add to the grid
+ */
+function addLetterToWord(letter) {
+    // Every letter should be uppercase
+    letter = letter.toUpperCase()
+
+    // Test if the received letter is accepted
+    if ("ABCDEFGHIJKLMNOPQRSTUVWXYZ".includes(letter)) {
```

Comment:
```suggestion
    if ("ABCDEFGHIJKLMNOPQRSTUVWXYZ".split("").includes(letter)) {
```
To prevent the catch of multiple "word" like `TUV` or `DEF`